src/standalone/Vegetation/Canopy.jl

```diff
@@ -77,6 +78,9 @@ fluxes.
 - The radiative flux conditions, which are either prescribed
 (of type `PrescribedRadiativeFluxes`) or computed via a coupled simulation
 (of type `CoupledRadiativeFluxes`).
+- The soil conditions, which are either prescribed (of type 'PrescribedSoil') or 
+fully or partially computed via a coupled simulation (of type 
```

Comment:
clarify a little?
"The soil conditions, which are either prescribed (of type `PrecribedSoil`, for running the canopy model in standalone mode), or prognostic (of type `PrognosticSoil`, for running integrated soil+canopy models)"